starter-code/finished_css_code

```diff
@@ -0,0 +1,113 @@
+*{
+  width: 850px;
```

Comment:
Rather than using a px value, use a % for width or set margin/padding so you can keep your site responsive.